ocaml/graph/README.markdown

```diff
@@ -10,24 +9,27 @@ To create a graph showing relationships between
 
 run:
 
-  $ graph -h <host> -u root -pw <password> VM VDI VBD SR > graph.gv
+```bash
+graph.native -h <host> -u root -pw <password> VM VDI VBD SR > graph.dot
+```
 
-install graphviz and then run:
+Then install graphviz and run:
 
-  $ dot -Tpng graph.gv -o graph.png
+```bash
+dot -Tsvg graph.dot -o graph.svg
+```
 
 Here's an example:
 
-![Example rendered graph](../../../../raw/master/ocaml/graph/example.png)
+![Example rendered graph](vm-disks.svg)
```

Comment:
Actually I could simply remove the image completely, since the user has instructions how to produce it and it's very easy to do so - this would solve the issue completely.